api/api.yaml

```
@@ -0,0 +1,120 @@
+openapi: 3.0.0
+info:
+  description: "Mainflux Homing Server"
+  version: 0.0.1
+  title: ET
+  termsOfService: ""
+  contact:
+    email: "infos@mainflux.com"
+  license:
+    name: Apache 2.0
+    url: https://raw.githubusercontent.com/mainflux/callhome/f5bef94fec5f1d1e35333fbde959fe1eeb42d944/LICENCE
```

You can change this to something like
```suggestion
    url: https://github.com/mainflux/callhome/blob/master/LICENSE
```